blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -132,6 +133,21 @@ namespace blueprint
                 View::paint(g);
             }
         }
+        //==============================================================================
+        bool keyPressed(const juce::KeyPress& key) override
+        {
+            if (jsDebugEnabled)
```

Comment:
Maybe we just `#if JUCE_DEBUG` this stuff and omit the `enableJSDebug()` method? I feel like it's fair to just always turn this on when in debug mode?

Reply:
Yeah I was going to ask exactly this actually! Nice. Will do so.

Reply:
@nick-thompson, in which case should ReactApplicationRoot call setWantsKeyboardFocus() in it's constructor? Seems like we probably want to make that the standard?

Reply:
Yea, that sounds good to me